templates/azure-functions/package.json

```
@@ -0,0 +1,25 @@
+{
+  "scripts": {
+    "build": "remix build",
+    "dev": "swa start ./public --run \"remix dev\" --api-location .",
+    "postinstall": "func init --worker-runtime node --language typescript"
+  },
+  "dependencies": {
+    "@remix-run/azure-functions": "*",
+    "@remix-run/react": "*",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "@azure/functions": "^3.0.0",
+    "@azure/static-web-apps-cli": "^0.8.3",
```

Shouldn't these be in `dependencies` as well? 🤔

```suggestion
    "@azure/functions": "^3.2.0",
    "@azure/static-web-apps-cli": "^1.0.2",
```

---

No. The `@azure/functions` package is just a types package and the CLI is only needed for local dev.